### PR TITLE
Add Editorial for Workshop CFP

### PR DESCRIPTION
**Pull request by vipulgupta2048**
Reviews needed @pythonindia/pycon-india-content-team @Pradhvan @ananyo2012 
Came up with this using PyCon US's Tutorials CFP as an inspiration. Please do read it once. 
Thanks

### CODE REVIEW

File: content/blog/2020-workshop-call-for-proposals.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+### Speaker Info
+
+You'll be asked to enter your biography and the information will be made public. Here, you may wish to include more information for reviewers to complement information about your previous experience. This can include your subject matter experience, past workshops if any, past speaking events if any, or anything else that would help us understand and gauge your abilities as a workshop presenter. A link to a video would be ideal (to be added in speaker URLs)
+
+In the end, we would like to mention the workshop selection process attempts to choose a good variety of workshop topics and levels that will be in demand, i.e., it doesn't just select the best proposals! The selection process has also tended to prefer Python-related topics over more general software development tools or practices that aren't Python-specific.
+
+
+## Where to submit your proposal
+
+Read the [call for proposals editorial](https://in.pycon.org/cfp/2020/proposals/) for more information on writing proposals. Next, start with drafting your proposal in the [**New Proposal**](https://in.pycon.org/cfp/2020/proposals/create/) page after signing up for an account.
```
Comment (ananyo2012) [inline content/blog/2020-workshop-call-for-proposals.md:58]: Please mention to select Workshops from the Proposal type Dropdown.

**Comment by ananyo2012** [comment]
@vipulgupta2048 Can you rebase your changes over latest master so that I can see the preview ?

**Comment by vipulgupta2048** [comment]
@ananyo2012 Added all your suggestions, I think this should cover it.

File: content/blog/2020-workshop-call-for-proposals.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+Slug: 2020-workshop-cfp-announcement
+
+The [call for proposals](https://in.pycon.org/cfp/2020/proposals/) for workshops has been officially opened! If you are slight to moderately unsure of submitting a workshop proposal, you won't be after reading this.
+
+<!-- PELICAN_END_SUMMARY -->
+Here's a question for you.
+
+Do you think that you would be able to assemble 3 hours of material and lead an online session full of PyCon attendees through learning a new skill, library, or technology? Then you might be someone we are looking for. Apply to host your workshop at PyCon India 2020! PyCon India 2020 workshops will be held online on October 4th, 2020.
+
+As with the talks, we are looking for workshops that can grow this community at any level. We aim for workshops that will advance Python, advance this community, and shape the future. Each workshop runs for 3 full hours plus a break for coffee. As workshops are longer, more in-depth than talks, they require much more preparation. Much more when it comes to writing your proposal.
```
Comment (ananyo2012) [inline content/blog/2020-workshop-call-for-proposals.md:14]: @anistark Can you quickly confirm if this is 2.5 hrs or 3 hrs ?

File: content/blog/2020-workshop-call-for-proposals.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+Author: Vipul Gupta (@vipulgupta2048)
+Category: 2020
+Slug: 2020-workshop-cfp-announcement
+
+The [call for proposals](https://in.pycon.org/cfp/2020/proposals/) for workshops has been officially opened! If you are slight to moderately unsure of submitting a workshop proposal, you won't be after reading this.
+
+<!-- PELICAN_END_SUMMARY -->
+Here's a question for you.
+
+Do you think that you would be able to assemble 3 hours of material and lead an online session full of PyCon attendees through learning a new skill, library, or technology? Then you might be someone we are looking for. Apply to host your workshop at PyCon India 2020! PyCon India 2020 workshops will be held online on October 4th, 2020.
```
Comment (ananyo2012) [inline content/blog/2020-workshop-call-for-proposals.md:12]: Please bold the date. Also confirm the duration is it 2.5 hrs or 3 hrs ?

**Comment by vipulgupta2048** [comment]
Thanks @ananyo2012 for the assist 🌟

**Pull request merged**